README.md

```diff
@@ -140,6 +140,45 @@ For a fair comparison, [LogLevel](http://npm.im/loglevel) was extended
 to include a timestamp and [bole](http://npm.im/bole) had
 `fastTime` mode switched on.
 
+<a name="transports"></a>
+## Transports
+
+A transport in most logging libraries is something that runs in-process to
+perform some operation with the finalized log line. For example, a transport
+might send the log line to a standard syslog server after processing the log
+line and reformatting it. For details on implementing, and some already written,
+transports, see our [Transports⇗](docs/transports.md) document.
+
+> **Pino *does not* natively support in-process transports.**
+
+Pino does not support in-process transports because Node processes are
+single threaded processes (ignoring some technical details). In order for Pino
+to achieve the speed it achieves, Pino purposefully offloads the handling of
+logs, and their ultimate destination, to external processes so that the
+threading capabilities of the OS can be used (or other CPUs).
+
+One consequence of this methodology is that "error" logs do not get written to
```

Comment:
I specifically left log4j out of this because what you are saying about log4j isn't actually true. We only follow the API it defines (`.error`, `.debug`, etc). But that's only half(?) of log4j. The other part of log4j is that it handles log management. Look at the log4j configuration for Elastic Search -- https://github.com/elastic/elasticsearch/blob/847378a43b066b33f731e4948589f6040f69d2f6/distribution/src/main/resources/config/log4j2.properties . All of that mess is configuring the log4j library to write the combined stream out to various destinations. So by saying we conform to the log4j standard, we'd be suggesting that we should support writing to multiple streams by default.